Library/Formula/platypus.rb

```diff
@@ -49,7 +40,7 @@ def install
   end
 
   test do
-    system "#{bin}/platypus", "-v"
+    system bin/"platypus", "-v"
```

Comment:
OK, I changed it back.  That wasn't the cause of the test weirdness with HEAD, though.